Proofreading about.rst and history.rst (#1709)
vyos/vyos-documentation: docs/introducing/about.rst

> VyOS is an open source network operating system based on Debian GNU/Linux.

Reviewer: ```suggestion
VyOS is an open-source network operating system that provides a single unified CLI and API to manage routing protocols, firewall and NAT, QoS, load balancing, DHCP and DNS servers, and many other features.
```

The underlying distro was probably never important to mention, now that I think of it. Users don't interact with VyOS as they would with a plain Debian.

Also, Debian developers renamed the Debian flavor that people actually use from "Debian GNU/Linux" to just "Debian" (Debian GNU/kFreeBSD was officially discontinued in 2023, Debian GNU/Hurd technically exists, but the GNU userland plus Linux kernel distro that everyone uses is just "Debian" now).

Author: Applied the suggested change via batch commit. Thank you.

> VyOS provides a free routing platform that directly competes with commercial 

Reviewer: This was written when VyOS was really small and unknown and also didn't provide any commercial services. We aren't exactly as big as Cisco or Juniper but I think the line is not needed anymore at all — we have a whole bunch of paying customers.

Author: Removed.

solutions from well-known network providers. Since VyOS runs on standard amd64, 

> i586, and ARM systems, it can be used as a router and firewall platform for 

Reviewer: VyOS doesn't run on i568 — we dropped support for it many years ago but neglected updating this section. There's also no mainline support for ARM(64) yet...

I think this line should be: "VyOS runs on a wide variety of commodity hardware, virtual machines, and multiple cloud environments".

Author: Updated.

cloud deployments.

> Beginning with VyOS 1.2, we provide a dedicated user guide for each major 

Reviewer: ```suggestion
We provide a dedicated user guide for each major 
```

It was important to mention the first version that had its own manual when that was a new thing for VyOS, but 1.2.0 was five years ago... I think now we can just say that manuals are versioned.

Author: Applied the suggested change via batch commit. Thank you.

VyOS release that receives long-term support (LTS). We maintain multiple user 

> guide versions, all hosted on Read the Docs https://readthedocs.org. 

Reviewer: ```suggestion
guide versions, all hosted at https://docs.vyos.io. 
```

We may move away from ReadTheDocs, and it's an implementation detail in any case. It's better to show an actual URL here.

Author: Applied the suggested change via batch commit. Thank you.

> The user guide version corresponds to its Git branch name. To switch between 

Reviewer: The version selection menu doesn't mention codenames anymore, only version numbers like 1.4 and 1.3. I think we can remove this passage. There was a point when it _only_ mentioned branch names and not numeric versions, that's probably why this passage exists.

Author: Removed.
